index.php

```diff
@@ -56,8 +56,10 @@
 
 	// Initialize Flux.
 	Flux::initialize(array(
-		'appConfigFile'      => FLUX_CONFIG_DIR.'/application.php',
-		'serversConfigFile'  => FLUX_CONFIG_DIR.'/servers.php',
+		'appConfigFile'           => FLUX_CONFIG_DIR.'/application.php',
+		'serversConfigFile'       => FLUX_CONFIG_DIR.'/servers.php',
+		'appConfigFileImport'     => FLUX_CONFIG_DIR.'/import/application.php',
+		'serversConfigFileImport' => FLUX_CONFIG_DIR.'/import/servers.php',
```

Comment:
Comment these two out if the performance hit of two extra `stat` calls are _that_ bad.